gateway/src/main/java/com/hurence/webapiservice/http/grafana/modele/AnnotationRequestParam.java

```diff
@@ -0,0 +1,178 @@
+package com.hurence.webapiservice.http.grafana.modele;
+
+import com.hurence.webapiservice.modele.AGG;
+import com.hurence.webapiservice.modele.SamplingConf;
+import com.hurence.webapiservice.timeseries.AnnotationRequest;
+import com.hurence.webapiservice.timeseries.TimeSeriesRequest;
+import io.vertx.core.json.Json;
+import io.vertx.core.json.JsonArray;
+
+import java.util.List;
+
+public class AnnotationRequestParam implements AnnotationRequest {
+
+    private JsonArray tags;
+    private long from;
+    private long to;
+    private long fromRaw;
+    private long toRaw;
+    private int maxAnnotation;
+    private Boolean matchAny;
+    private String type;
+
+
+    private AnnotationRequestParam() { }
+
+    public void setTags(JsonArray tags) {
+        this.tags = tags;
+    }
+
+    public void setFrom(long from) {
+        this.from = from;
+    }
+
+    public void setTo(long to) {
+        this.to = to;
+    }
+
+    public void setFromRaw(long fromRaw) {
+        this.fromRaw = fromRaw;
+    }
+
+    public void setToRaw(long toRaw) {
+        this.toRaw = toRaw;
+    }
+
+    public void setMaxAnnotation(int maxAnnotation) {
+        this.maxAnnotation = maxAnnotation;
+    }
+
+    public void setMatchAny(Boolean matchAny) {
+        this.matchAny = matchAny;
+    }
+
+    public void setType(String type) {
+        this.type = type;
+    }
+
+    public JsonArray getTagsAsJsonArray() { // i need to use this getTags without the one in the TimeSeriesRquest
```

Comment:
What does mean this comment ? What is the relation here with TimeSeriesRequest ?